odoo/fields.py

```diff
@@ -1683,6 +1683,7 @@ class Html(_String):
     """
     type = 'html'
     column_type = ('text', 'text')
+    column_cast_from = ('varchar',)
```

Comment:
i don't know how you plan to rebase your commits, but I think it'd make more sense to have this change in the feature commit

Reply:
True, I probably have this one here because it was the first commit where I needed to replace a `Char` field with an `Html` one.